docs/docs/index.md

```diff
@@ -9,6 +9,8 @@ To continue, please select one of the following options:
 
 - I am a user of the platform or interested in its features:
   [User documentation](./user/index.md)
+- If you want to automate tasks using our API, have a look at the
+  [API Documentation](./api/index.md)
```

Comment:
The pattern seems to be "I am a..." so maybe:

I am a user of the platform interested in automating my tasks or integrating with other systems: API blah